src/TaskModel.vala

```diff
@@ -272,12 +276,11 @@ public class Tasks.TaskModel : Object {
                         var webdav_discovered_source = webdav_discovered_sources.nth_data (0);
                         webdav_server_uri = new Soup.URI (webdav_discovered_source.href.dup ());
                     }
-                    /**
-                     * TEMPORARY WORKAROUND: `E.webdav_discover_do_free_discovered_sources`
-                     * Remove this line, once the following commit of libedataserver is released:
-                     * https://gitlab.gnome.org/GNOME/evolution-data-server/-/commit/9d1505cd3518ff32bd03050fd898abf89d31d389
-                     */
+
+#if HAS_EDS_3_40
+#else
```

Comment:
```suggestion
#if !HAS_EDS_3_40
```

Reply:
wasn't aware this is possible - thanks for the hint!